src/modules/client/components/planning/Agenda.vue

```diff
@@ -121,8 +125,19 @@ export default {
     openEventCreationModal (value) {
       this.$emit('open-creation-modal', value);
     },
-    openEventEditionModal (value) {
-      this.$emit('open-edition-modal', value);
+    openEventInfoModal (event, conflictingEvents) {
+      const value = this.isCustomerPlanning ? conflictingEvents : event;
+      this.$emit('open-info-modal', value);
+    },
+    getConflictEvents (event, dayIndex) {
+      const dailyEvents = this.getOneDayEvents(this.days[dayIndex]);
+      const conflictingEvents = dailyEvents.filter(
+        ev => this.$moment(event.startDate).isBetween(ev.startDate, ev.endDate) ||
+          this.$moment(ev.startDate).isBetween(event.startDate, event.endDate) ||
+          this.$moment(event.endDate).isBetween(event.startDate, event.endDate) ||
+          this.$moment(ev.endDate).isBetween(event.startDate, event.endDate)
```

Comment:
je crois que tu peux simplifier en seulement 2 conditions
this.$moment(event.startDate).isBetween(ev.startDate, ev.endDate) ||
this.$moment(event.endDate).isBetween(event.startDate, event.endDate) ||